react-api-client/src/runs/useAddCapturePreviewImageToRun.ts

```diff
@@ -0,0 +1,58 @@
+import { useMutation, useQueryClient } from 'react-query'
+
+import { addCapturePreviewImageToRun } from '@opentrons/api-client'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type { UseMutateFunction, UseMutationResult } from 'react-query'
+import type {
+  CameraImageSettings,
+  DownloadedPreviewImageFileResponse,
+  ErrorResponse,
+} from '@opentrons/api-client'
+
+export interface AddCapturePreviewImageToRunParams {
+  runId: string
+  settings: CameraImageSettings
+}
+
+export type UseAddCapturePreviewImageToRunMutationResult = UseMutationResult<
+  DownloadedPreviewImageFileResponse,
+  AxiosError<ErrorResponse>,
+  AddCapturePreviewImageToRunParams
+> & {
+  addCapturePreviewImageToRun: UseMutateFunction<
+    DownloadedPreviewImageFileResponse,
+    AxiosError<ErrorResponse>,
+    AddCapturePreviewImageToRunParams
+  >
+}
+export function useCapturePreviewImageToRun(): UseAddCapturePreviewImageToRunMutationResult {
```

Comment:
```suggestion
export function useCapturePreviewImageToRun(runId: string): UseAddCapturePreviewImageToRunMutationResult {
```
Hm, I think we save ourselves a lot of grief and make our code a bit more consistent with other hooks if we define the `runId` at the hook-level scope rather than the internal function scope.